### ISSUE TITLE
setRippleEnabled from code

### ISSUE THREAD

**Issue by FunnyDevs** [H07000223/FlycoRoundView#10]
Hi! Is possible to have rippleenabled even from java??? Thanks

**Comment by ysmintor** [comment]
Agree, I think author @H07000223 might add material design ripple effect into this view. Hope update!
